src/components/Dashboard/TheSidebar.vue

```diff
@@ -1,58 +1,47 @@
 <template>
   <b-menu>
-    <b-menu-list>
+    <b-menu-list
+      v-for="menuListItem in groupMenuList"
+      :key="menuListItem.group"
+      :label="menuListItem.label"
+    >
       <b-menu-item
-        :to="{ name: 'backoffice.dashboard' }"
+        v-for="menuItem in menuListItem.menu_items"
+        :key="menuItem.to.name"
+        :to="menuItem.to"
         tag="router-link"
-        icon="chart-line"
-        label="View Dashboard"
-      ></b-menu-item>
-    </b-menu-list>
-
-    <b-menu-list label="Triage">
-      <b-menu-item
-        :to="{ name: 'backoffice.triageNew' }"
-        tag="router-link"
-        icon="plus"
-        label="Admit a patient"
-      ></b-menu-item>
-    </b-menu-list>
-
-    <b-menu-list label="Admitted Patients">
-      <b-menu-item
-        :to="{ name: 'backoffice.admissionsList' }"
-        tag="router-link"
-        icon="bed"
-        label="Admitted patients"
-      ></b-menu-item>
-    </b-menu-list>
-
-    <b-menu-list label="Bed Management">
-      <b-menu-item
-        :to="{ name: 'backoffice.bedTypeList' }"
-        tag="router-link"
-        icon="bed"
-        label="See all bed types"
-      ></b-menu-item>
-      <b-menu-item
-        :to="{ name: 'backoffice.bedList' }"
-        tag="router-link"
-        icon="bed"
-        label="See all beds"
-      ></b-menu-item>
-    </b-menu-list>
-
-    <b-menu-list label="Health Snapshots">
-      <b-menu-item
-        :to="{ name: 'backoffice.healthSnapshotNew' }"
-        tag="router-link"
-        icon="plus"
-        label="Create a health snapshot"
+        :icon="menuItem.icon"
+        :label="menuItem.label"
       ></b-menu-item>
     </b-menu-list>
   </b-menu>
 </template>
 
 <script>
-export default {};
+import { mapState } from "vuex";
+export default {
+  computed: {
+    ...mapState(["sidebarmenu", "user", "patient", "admission"]),
+    groupMenuList: function() {
+      // Filter the sidebarMenuList for the user's groups
+
+      var list = this.sidebarmenu.sidebarMenuList.filter(
+        listitem => this.user.groups.indexOf(listitem.group) >= 0
```

Comment:
Not that important but just in case you didn't know: you can now use `Array.prototype.includes()` ;)

[MDN Doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
Yeah, need to brush up on my JS. Thanks for the reminder :smile: